eng/source-build-release-announcement.md

```diff
@@ -0,0 +1,5 @@
+[Release Notes]($RELEASE_NOTES_URL) | [Tag]($TAG_URL)
```

Comment:
Is it feasible to link to the dotnet/core announcement issue - e.g. https://github.com/dotnet/core/issues/7259?

Maybe something to log as a follow-up issue.  Linking to the discussion of the issues customers are encountering is valuable here.

Like the dotnet/core issue, I think we should point folks where to log issues.